### DISCUSSION TITLE
Courses still showing In Progress

### DISCUSSION THREAD

**Discussion by vkg02283** [github/india#575] [Students]
@ey-mentor1 @ey-mentor2 @EY-Mentor3 @EY-Mentor4
Sir, I have completed both the following courses:
Azure Fundamentals (AZ-900)
Azure Administrator (AZ-104)
and it is still showing in progress from a long time.
Here is my MS Learn id-https://docs.microsoft.com/en-us/users/vivekkumar-8845/
Please resolve my issue. 
Thanking you
Yours Sincerely
Vivek Kumar

**Comment by Ruchita-jain** [comment]
I also have the same problem.

There is problem in future skills prime because I completed my course in Microsoft learn. I got my full XP but in future skill prime it is not showing the button of complete. So my course is completed but it is still showing in progress. what do I have to do now??

Here is my MS Learn profile- https://docs.microsoft.com/en-us/users/ruchitajain-4559/

**Reply by AmanJulka** [reply]
I have the same issue! 
Microsoft Support team Please help us???
